### ISSUE TITLE
Trouble attaching screenshot

### ISSUE THREAD

**Issue by msydejes** [reportportal/reportportal#799]
**Describe the bug**
Hi, I am currently having trouble getting screenshots to show up in report portal. 

**Expected behavior**
My code currently looks like this

```
          SaveLogRQ logrq = new SaveLogRQ();
          logrq.setLevel("error");
          logrq.setMessage(failure.getMessage());
          logrq.setTestItemId(testCaseId.blockingGet());
          logrq.setLogTime(new Date());
          String screenshotLocal = "C:\\Users\\msydejesus\\server\\test.png";

          try {
            File file= new File(screenshotLocal);

            SaveLogRQ.File screenshotFile = new SaveLogRQ.File();
            screenshotFile.setContentType(detect(file));
            screenshotFile.setContent(toByteArray(file));

            screenshotFile.setName(UUID.randomUUID().toString());
            logrq.setFile(screenshotFile);

          } catch (IOException e) {
            throw e;
          }

          Maybe<EntryCreatedRS> response = this.reportPortal.getClient().log(logrq);

```

The failure appears properly in the test case, but there are no attachments to be seen.
![image](https://user-images.githubusercontent.com/25910131/72762551-dc925e80-3b94-11ea-981c-e5851fcd656c.png)

**Versions:**
 - Windows 10, Chrome

**Comment by evjlobanova** [comment]
@msydejes please check  

https://github.com/reportportal/client-java/blob/7806432846add7935a4171ad921e4648006c888a/src/main/java/com/epam/reportportal/service/ReportPortal.java#L172

**Comment by evjlobanova** [comment]
Also we have released a new version 5.0

**Comment by evjlobanova** [comment]
@msydejes do you still have a problem?
If it still reproduces, feel free to reopen
